CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ export WORKDIR=~/gits/rapidsai/cudf
 Currently libcudf is only configured once and the build relies on cmake to re-configure as needed.
 This is because libcudf currently is rebuilding almost entirely when it is configured with the same
 settings. If an explicit reconfigure of libcudf is needed (e.g.: when changing compile settings via
-`GPU_ARCHS`, `CUDF_USE_PER_THREAD_DEFAULT_STREAM`, etc.) then a configure can be forced via
```

Comment:
I think we should deprecate it first, rather than remove it. I see at least ~200 chat messages internally referencing it.

Reply:
Sure, I'll do that.

Reply:
Done. Please take a another look.